packages/app-data/src/api/fetchDocFromAppData.ts

```
@@ -0,0 +1,53 @@
+ * Fetches the document from IPFS using the appData hex
+ *
+ * @deprecated Uses the legacy method of deriving the CID from the appData hex
+ *
+ * @param appDataHex
+ * @param ipfsUri
+ * @returns
+ */
+export async function fetchDocFromAppDataHexLegacy(
+  appDataHex: string,
+  ipfsUri?: string
+): Promise<void | AnyAppDataDocVersion> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)